assets/chat/js/formatters.js

```diff
@@ -612,7 +612,7 @@ class UrlFormatter {
             relaxed = strict + "|" + webURL;
         this.linkregex = new RegExp(relaxed, "gi");
         this.discordmp4Regex = new RegExp("(https?:\/\/)?(www\.)?(cdn.discordapp\.com)?\.(mp4|webm)")
-
+        this.refLinkRegex = new RegExp("(https?://)?(www.)?(amazon)?(twitter)?(spotify)?");
```

Comment:
do you need to escape `/` and `.` like in the discordmp4 `RegExp`?

Reply:
This is an outdated PR >.< Remaking it with updated changes, sorry